### PR TITLE
Feature/cs4360 361 ng account service

### PR DESCRIPTION
**Pull request by GitNickProgramming**

### CODE REVIEW

File: pb-crm-ui/src/app/invoicing/services/AccountService.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    return this.apiService.post<CreateAccountResponse>('/api/contact', req, {}).toPromise();
+  }
+
+  async updateAccount(account: AccountModel): Promise<UpdateAccountResponse>{
+    const req: UpdateAccountRequest = { account }; // cast will fail if invalid model passed
+    return this.apiService.put<UpdateAccountResponse>('/api/contact/' + account.id, req, {}).toPromise();
+  }
+
+  deleteAccount(id: string): Observable<any> {
+    const url = '/api/account/' + id;
```
Comment (Newms226) [inline pb-crm-ui/src/app/invoicing/services/AccountService.ts:39]: This route is correct

File: pb-crm-ui/src/app/invoicing/services/AccountService.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+  async createAccount(account: AccountModel): Promise<CreateAccountResponse> {
+    const req: CreateAccountRequest = { account };
+    return this.apiService.post<CreateAccountResponse>('/api/contact', req, {}).toPromise();
+  }
+
+  async updateAccount(account: AccountModel): Promise<UpdateAccountResponse>{
+    const req: UpdateAccountRequest = { account }; // cast will fail if invalid model passed
+    return this.apiService.put<UpdateAccountResponse>('/api/contact/' + account.id, req, {}).toPromise();
+  }
```
Comment (Newms226) [inline pb-crm-ui/src/app/invoicing/services/AccountService.ts:36]: These routes are all still wrong. Notice how you are trying to talk to `/api/contact` not `api/account`

Comment (GitNickProgramming) [reply pb-crm-ui/src/app/invoicing/services/AccountService.ts:36]: derp didnt see that

**Comment by GitNickProgramming** [comment]
fixing the derp issues

**Pull request closed without merging**